### PR TITLE
03 add navigation pokemons infos

### PR DESCRIPTION
**Pull request by RobinDela**

### CODE REVIEW

**Review by AntoineCAU — DISMISSED** [review]
Perfect, i understand everything.
Good feature

File: src/components/PokemonPage.jsx [hunk 1]
```diff
@@ -82,6 +82,14 @@ const PokemonPage = () => {
     }
   };
 
+  const handleEventNext = () => {
+    history.push(`/id/${infos.id + 1}`);
+  };
+
+  const handleEventPrevious = () => {
+    history.push(`/id/${infos.id - 1}`);
```
Comment (LoicBrassart) [inline src/components/PokemonPage.jsx]: Beware of edge cases: what happens if I'm on Bulbasaur (#1) ? Or on the last one ?

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
Edge cases to fix

**Review by AntoineCAU — DISMISSED** [review]
Perfect Robin, nice fix

**Review by LoicBrassart — DISMISSED** [review]
Much better !

**Review by AntoineCAU — DISMISSED** [review]

**Review by LoicBrassart — APPROVED** [review]
No more Evil

**Review by AntoineCAU — APPROVED** [review]

**Pull request merged**